src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -86,6 +86,14 @@ public void Add(CookieCollection cookies)
             }
         }
 
+        public bool IsReadOnly
+        {
+            get
+            {
+                return false;
```

Comment:
@CIPop @stephentoub ReadOnly false seems to be the logical choice here and that's why i left it as false. Opened the issue https://github.com/dotnet/corefx/issues/12959 if we need to fix it later.